utils/auth/auth.js

```diff
@@ -0,0 +1,22 @@
+import jwt from "jsonwebtoken";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+/**
+ *  Authenticate users
+ */
+module.exports = function generateToken(_id, email) {
```

Comment:
This should be `generateToken(_id, email)` `loosely coupled` not `tightly coupled` => `generateToken(payload = {} )`also why are you using email as payload remeber user can their email address there for is not consistent.